RealmSwift/Realm.swift

```diff
@@ -1192,35 +1204,54 @@ extension Realm {
     @MainActor
```

Comment:
Does this mean that we are still constrained to the  the main actor to call try await Realm() ?

Reply:
Yes, this does not add support for confining Realms to arbitrary actors.